main.tf

```diff
@@ -110,7 +110,10 @@ resource "aws_cloudwatch_log_group" "this" {
   retention_in_days = var.cloudwatch_log_group_retention_in_days
   kms_key_id        = var.cloudwatch_log_group_kms_key_id
 
-  tags = var.tags
+  tags = merge(
```

Comment:
If you define `Name` in var.tags, `Name = "/aws/eks/${var.cluster_name}/cluster"` will be replaced by value in var.tags.
```
tags = merge(
  var.tags,
  { Name = "/aws/eks/${var.cluster_name}/cluster" }
)
```
I think it will be better.

Reply:
https://developer.hashicorp.com/terraform/language/functions/merge#examples

Reply:
Great suggestion. Fixed.